add-on/src/main/java/org/vaadin/crudui/form/AbstractAutoGeneratedCrudFormFactory.java

```diff
@@ -318,10 +318,11 @@ public void showError(CrudOperation operation, Exception e) {
         } else {
             if (CrudOperationException.class.isAssignableFrom(e.getClass())) {
                 // FIXME no Notification.Type
-                Notification.show(e.getMessage());
+                Notification.show(e.toString());
             } else {
-                Notification.show(e.getMessage());
-                throw new RuntimeException("Error executing " + operation.name() + " operation", e);
+                String msg = "Error executing " + operation.name() + " operation";
+                Notification.show(msg + "\n" + e.toString());
```

Comment:
I would prefer a null check. I'll fix it right away. Thanks for your contribution!